bundle/ios/GutenbergNativeTranslations.swift

```diff
@@ -157,8 +157,10 @@ private func dummy() {
     _ = NSLocalizedString("Featured Image", comment: "")
     _ = NSLocalizedString("File block settings", comment: "")
     _ = NSLocalizedString("File name", comment: "")
+    _ = NSLocalizedString("Font Size", comment: "")
```

Comment:
- Addition
- [Code reference](https://github.com/WordPress/gutenberg/blob/trunk/packages/components/src/font-size-picker/index.native.js#L59:L59)

Looks like we missed this string, most likely this was caused due to the parsing issues when extracting the strings.